indigo_api/serializers.py

```diff
@@ -653,11 +653,26 @@ def update(self, work, validated_data):
             # this is really a Country object
             validated_data['country'] = validated_data['country']['code']
 
+        # ensure any docs for this work at initial pub date move with
+        ## get old date
+        old_date = work.publication_date
+
+        ## get docs
+        docs = Document.objects.filter(work=self.instance, expression_date=old_date)
+
         # save as a revision
         with reversion.revisions.create_revision():
             reversion.revisions.set_user(user)
             work = super(WorkSerializer, self).update(work, validated_data)
 
+        ## get new date
+        new_date = work.publication_date
+
+        ## change and save docs once work has successfully updated
+        for doc in docs:
```

Comment:
I think we should only do this if the date doesn't match -- saving a document isn't "cheap".